File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/multiline/uncomment/direct/Java/Python/s824230885_commented.py
def main():
    import sys
    def next_int():
        return int(sys.stdin.readline())
    def next_array():
        return list(map(int, sys.stdin.readline().split()))
    N = next_int()
    arr = next_array()
    max_val = 0
    output = 0
    for num in arr:
        if max_val <= num:
            max_val = num
            output += 1
    print(output)

if __name__ == "__main__":
    main()
# End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/multiline/uncomment/direct/Java/Go/s824230885_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	nStr, _ := reader.ReadString('\n')
	n, _ := strconv.Atoi(strings.TrimSpace(nStr))

	arrStr, _ := reader.ReadString('\n')
	arrStr = strings.TrimSpace(arrStr)
	parts := strings.Split(arrStr, " ")

	var arr []int
	for _, p := range parts {
		num, _ := strconv.Atoi(p)
		arr = append(arr, num)
	}

	max := 0
	output := 0
	for _, num := range arr {
		if num >= max {
			max = num
			output++
		}
	}

	fmt.Println(output)
}

// End of Code
